zero/tok_geo.c

```diff
@@ -597,6 +597,11 @@ gkyl_tok_geo_get_stat(const gkyl_tok_geo *geo)
 void
 gkyl_tok_geo_release(gkyl_tok_geo *geo)
 {
+  gkyl_array_release(geo->psiRZ);
+  gkyl_array_release(geo->psibyrRZ);
+  gkyl_array_release(geo->psibyr2RZ);
+  gkyl_array_release(geo->fpoldg);
+  gkyl_array_release(geo->qdg);
```

Comment:
@akashukla likewise, are these bugs @Maxwell-Rosen found in the mirror branch? are there similar bugs in mirror_geo?